### PR TITLE
Update numeric.md

### PR DESCRIPTION
**Pull request by soerenwolfers**
This is part one - document the expected behavior - of my dissection of https://github.com/duckdb/duckdb/issues/14239, which I closed for being too chaotic.

### CODE REVIEW

File: docs/sql/data_types/numeric.md [hunk 1]
```diff
@@ -75,16 +75,25 @@ In addition to ordinary numeric values, the floating-point types have several sp
...
 * `NaN`: not a number
 
-> On a machine whose floating-point arithmetic does not follow IEEE 754, these values will probably not work as expected.
+On machines with the required CPU/FPU support, DuckDB follows the IEEE 754 specification regarding these special values, with two exceptions:
 
-When writing these values as constants in a SQL command, you must put quotes around them, for example:
+* `NaN` compares equal to `NaN` and greater than any other floating point number.
+* Some floating point functions, like `sqrt` / `sin` / `asin` throw errors rather than return `NaN` for values outside their ranges of definition.
+
+To insert these values as literals in a SQL command, you must put quotes around them, you may use any capitalization, and you may abbreviate `Infinity` as `Inf`. For example:
```
Comment (soerenwolfers) [inline docs/sql/data_types/numeric.md:83]:
```suggestion
To insert these values as literals in a SQL command, you must put quotes around them, you may abbreviate `Infinity` as `Inf`, and  you may use any capitalization. For example:
```

File: docs/sql/data_types/numeric.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ On machines with the required CPU/FPU support, DuckDB follows the IEEE 754 speci
 * `NaN` compares equal to `NaN` and greater than any other floating point number.
 * Some floating point functions, like `sqrt` / `sin` / `asin` throw errors rather than return `NaN` for values outside their ranges of definition.
 
-To insert these values as literals in a SQL command, you must put quotes around them, you may use any capitalization, and you may abbreviate `Infinity` as `Inf`. For example:
+To insert these values as literals in a SQL command, you must put quotes around them, you may abbreviate `Infinity` as `Inf`, and  you may use any capitalization. For example:
```
Comment (szarnyasg) [inline docs/sql/data_types/numeric.md:83]:
```suggestion
To insert these values as literals in a SQL command, you must put quotes around them, you may abbreviate `Infinity` as `Inf`, and you may use any capitalization. For example:
```

File: docs/sql/data_types/numeric.md [hunk 3]
```diff
@@ -75,16 +75,25 @@ In addition to ordinary numeric values, the floating-point types have several sp
...
 ```sql
-UPDATE table
-SET x = '-Infinity';
+SELECT
+    sqrt(2) > '-inf',
+    'nan' > sqrt(2)
 ```
 
-On input, these strings are recognized in a case-insensitive manner.
+<div class="narrow_table"></div>
```
Comment (szarnyasg) [inline docs/sql/data_types/numeric.md:91]:
```suggestion
<div class="narrow_table monospace_table"></div>
```

**Comment by szarnyasg** [comment]
Thanks!

**Comment by soerenwolfers** [comment]
when to use narrow and when monospace?

**Comment by szarnyasg** [comment]
I don't have a 100% consistent rule yet. I applied monospace because I thought it will apply to the headers here... but I was wrong. Will try to come up with a recommendation in the contributor guide.

**Pull request merged**